### PR TITLE
add assets from Picasso eth ibc connection

### PR DESCRIPTION
**Pull request by JafarAz**
### Adding Assets

If adding a new asset, please ensure the following:
- [x] Asset is registered to the [Cosmos Chain Registry](https://github.com/cosmos/chain-registry).
- [x] Add asset to bottom of `zone_assets.json`.
   - [x] `chain_name` and `base_denom` are provided and use values exactly as defined at the Chain Registry.
   - [x] `path` is provided, and the IBC channel referenced is registered at the Chain Registry (unless native to Osmosis).
   - [x] `osmosis_verified` is set to `false`
   - [x] `listing_date_time_utc` is specified and accurate (not required when adding as unlisted)
   - [ ] Optional: `transfer_methods`, `peg_mechanism`, `override_properties`, `canonical`, `categories`, where necessary (see [README](https://github.com/osmosis-labs/assetlists/tree/main?tab=readme-ov-file#how-to-add-assets) for details).

### CODE REVIEW

File: osmosis-1/osmosis-1.chainlist.json [hunk 1]
```diff
@@ -5548,7 +5548,7 @@
           }
         ]
       },
-      "description": "Composable is the base layer connecting L1s and L2s. We are scaling IBC to other ecosystems, pushing the boundaries of trust-minimized interoperability. We abstract the cross-chain experience for users, delivering seamless chain-agnostic execution of user intentions.",
```
Comment (JeremyParish69) [inline osmosis-1/osmosis-1.chainlist.json:5551]: the Chainlist should not be edited directly. Any changes of data should be done at the Chain Registry, where the automated script pulls this data from

File: osmosis-1/osmosis-1.assetlist.json [hunk 2]
```diff
@@ -9814,6 +9814,70 @@
         "osmosis-price:uosmo:1145"
       ]
     },
+    {
```
Comment (JeremyParish69) [inline osmosis-1/osmosis-1.assetlist.json:9817]: the Assetlist should not be edited directly. Any changes of data should be done at the Chain Registry, where the automated script pulls this data from

Comment (JeremyParish69) [reply osmosis-1/osmosis-1.assetlist.json:9817]: All added assets should go into the zone_assets file

Comment (JafarAz) [reply osmosis-1/osmosis-1.assetlist.json]: updated

File: osmosis-1/osmosis.zone_assets.json [hunk 3]
```diff
@@ -2162,13 +2162,77 @@
...
+          "name": "Mantis App",
           "type": "external_interface",
-          "deposit_url": "https://app.trustless.zone/?from=PICASSO&to=OSMOSIS",
-          "withdraw_url": "https://app.trustless.zone/?from=OSMOSIS&to=PICASSO"
+          "deposit_url": "https://games.mantis.app/?from=POLKADOT&to=OSMOSIS",
+          "withdraw_url": "https://games.mantis.app/?from=OSMOSIS&to=POLKADOT"
         }
       ]
     },
+    {
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_assets.json:2172]: Please add these at the bottom of the list of assets

File: osmosis-1/osmosis.zone_assets.json [hunk 4]
```diff
@@ -3641,6 +3641,70 @@
...
       "listing_date_time_utc": "2024-04-14T01:25:00Z",
       "_comment": "ReStake DAO $RSTK"
+    },
+    {
+      "chain_name": "composable",
+      "base_denom": "ibc/F9D075D4079FC56A9C49B601E54A45292C319D8B0E8CC0F8439041130AA7166C",
+      "path": "transfer/channel-1279/transfer/channel-52/wei",
+      "osmosis_unlisted": true,
+      "osmosis_verified": false,
+      "_comment": "Ethereum $ETH"
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_assets.json:3651]: Without a transferMethods section on these assets, the ibc transfers will by default just go to Composable, and not all the way to Ethereum. Is this the desired behaviour for Withdraw? (and vice versa for Deposit--can only deposit from Composable, but not able to deposit from Ethereum.)

My understanding was that Osmosis would be added as a destination network on Mantis to enable transfers across the full path between Ethereum and Osmosis.

Comment (JafarAz) [reply osmosis-1/osmosis.zone_assets.json]: ty

added this section, lmk if I am missing anything else

Comment (JeremyParish69) [reply osmosis-1/osmosis.zone_assets.json]: Not a strict requirement, but it would be nice if the URLs accepted parameters that preselected the asset.

Comment (JafarAz) [reply osmosis-1/osmosis.zone_assets.json]: Can I make this in a future PR if possible? Once the assets have been listed

**Pull request merged**